static/images/doubleleftarrow.svg

```diff
@@ -0,0 +1,43 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 400000 549" preserveAspectRatio="xMinYMin slice">
```

Comment:
Some of the SVGs are nested, so that the inner SVG can keep an undisturbed aspect ratio. For instance, we want an arrow head to be un-distorted when the arrow gets longer or shorter. In these files, each arrow SVG contains an arrow that is 400 ems long in the inner SVG. The outer SVG is set to match the dimensions defined for it by the CSS. The outer SVG clips the inner SVG; it does not stretch the inner SVG.

Of course, we do want some of the SVGs, like `\cancel`, to vary their aspect ratio. Those files do not get the nested SVG treatment.